_posts/2021-04-20-jump_dev_2021.md

```diff
@@ -26,47 +28,9 @@ registering for JuliaCon you can attend talks at JuMP-dev and JuliaCon.
 
 All participants will uphold the [JuliaCon Code of Conduct](https://juliacon.org/2021/coc/).
 
-### How to give a talk
-
-Similarly to [previous](/meetings/mit2017) [iterations](/meetings/bordeaux2018)
-of the [workshop](/meetings/santiago2019), we invite participants to present
-work related to JuMP. In particular, we especially seek talks related to:
-
-- JuMP core development ([MathOptInterface](https://github.com/JuliaOpt/MathOptInterface.jl), JuMP 1.0)
```

Comment:
I think it's useful to keep the details of the call for talks posted both for historical interest and for the benefit of people deciding if they want to attend.